api/python/provisioner/commands/__init__.py

```diff
@@ -536,13 +536,63 @@ def dynamic_validation(self, params: inputs.SWUpdateRepo, targets: str):
             #   - there is no the same release repo already active EOS-13715
             #   - (optionally) new version is higher then currently installed
 
+            cmd = ('yum --disablerepo="*" '
+                   f'--enablerepo="sw_update_{candidate_repo.release}" '
+                   'list available')
+            try:
+                salt_cmd_run(cmd, targets=LOCAL_MINION)
+            except SaltCmdRunError as e:
+                # SW Update repo is malformed
+                logger.debug("Can't enable SW update candidate repo: "
+                             f"'{str(e)}'")
+
+            # TODO: take it from pillar
+            iso_mount_dir = f'/opt/seagate/cortx/updates/{REPO_CANDIDATE_NAME}'
```

Comment:
Can this base directory be set in config.py?
There's a possibility for change regarding this path.

Reply:
There is pillar value for this directory. It is used for on-fly mount of ISO during validation and I can reuse it.
Added logic to retrieve this value from pillars